ocb3/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "ocb3"
+version = "0.1.0"
+description = """
+Pure Rust implementation of the AES-OCB3
+Authenticated Encryption with Associated Data (AEAD) Cipher
```

Comment:
I think we can remove the mention of AES and instead say that it's an implementation generic over block ciphers.